### ISSUE TITLE
申し訳ございません。このページに到達できません というエラーがでる

### ISSUE THREAD

**Issue by detsuku** [ElectronicObserverEN/ElectronicObserver#255]
こんにちは

表題のエラーメッセージがでてゲームができなくなりました。
更新(F5)、ログインページへ移動を行っても解消されません。
移動(移動先の入力)で出てくるurlをedgeに入力するとゲームができます。

5.1.13を手動上書きしても解消されませんでした。 

<img width="193" alt="スクリーンショット 2022-11-14 145347移動" src="https://user-images.githubusercontent.com/77409739/201593570-a991094e-48c9-4c92-87df-05e03c489e06.png">

**Comment by myangelkamikaze** [comment]
Can you make a screenshot of the error?
Did you try clearing the cache?
![image](https://user-images.githubusercontent.com/40002167/201597477-ac299259-cc57-4296-996b-3a7acdb0a76a.png)

**Comment by detsuku** [comment]
Attached is a screenshot of the error.
Clearing the cache did not solve the problem.
![20221114_17422122](https://user-images.githubusercontent.com/77409739/201615372-a1cb15de-7348-4c74-b67e-78cc22b510f3.png)

**Comment by myangelkamikaze** [comment]
Hmm, I don't think this is a problem with 七四式.
Since you're getting the direct url from somewhere else, I assume this doesn't happen on other browsers?
Can you try using a VPN and see if this problem still happens?
Also, you can write in Japanese if you want, it's not a problem.

**Comment by detsuku** [comment]
I tried VPN and saw no resolution.
The game is playable on edge.
The problem still occurs even with the bare zip extracted.

**Comment by myangelkamikaze** [comment]
What does your connection configuration look like?
![image](https://user-images.githubusercontent.com/40002167/201924134-64de6292-ec38-498a-8fda-5180c687444b.png)

**Comment by detsuku** [comment]
The original setting was 40620 for the port.
<img width="392" alt="スクリーンショット 2022-11-15 220411設定画面" src="https://user-images.githubusercontent.com/77409739/201931399-347f4cfb-6bdb-4114-b233-94965c42ca7d.png">

I tried setting it to the same value as in the screenshot, but that did not solve the problem.
<img width="351" alt="スクリーンショット 2022-11-15 220712設定画面2" src="https://user-images.githubusercontent.com/77409739/201931607-2a2d9bcd-ddf7-4cb0-b635-bf73685a25ef.png">

**Comment by myangelkamikaze** [comment]
The exact port numbers aren't really important, it was more to check if there's anything enabled that could be causing the issue.

Maybe try the solutions listed [here](https://answers.microsoft.com/en-us/microsoftedge/forum/all/errconnectionreset/ec4ac800-0c61-4a63-8778-65853c0a18e9)?

**Comment by detsuku** [comment]
I have troubleshooted my internet connection and could not identify the problem.

Specifying the url for the 移動(移動先の入力)) did not identify the problem.

I also tried entering the latter command, but it did not resolve the issue.

**Comment by detsuku** [comment]
Thank you very much.

Problem solved.

**Comment by myangelkamikaze** [comment]
Ah, good, I was running out of ideas w

**Comment by myangelkamikaze** [comment]
Can you try opening developer tools and then check what's going on in the network tab when the browser tries to go to the login page?

![image](https://user-images.githubusercontent.com/40002167/202696378-c7816026-fce1-4972-85cc-ce535e2ad0a4.png)
![image](https://user-images.githubusercontent.com/40002167/202696558-928e90d7-39ed-41e7-bbd4-e5267076c7ee.png)

**Comment by detsuku** [comment]
At first nothing was displayed.
After updating, the first line was displayed.
After navigating to the login page, the second and subsequent lines were displayed.

<img width="1280" alt="スクリーンショット 2022-11-19 031426開発者モード" src="https://user-images.githubusercontent.com/77409739/202776249-d5304e6d-ffb6-48d8-a96e-5a4d28e4bd36.png">

**Comment by detsuku** [comment]
I am not using a proxy.
So it seems that we need to try to initialize or reinstall the OS.
I need to prepare for that as well, so it will take some time to report back.

**Comment by myangelkamikaze** [comment]
Might be a good idea to try a different computer first.
Maybe a different windows account too.

**Comment by myangelkamikaze** [comment]
Looks like somebody else has the same problem on Twitter, so I can only think of WebView2 being the problem.
https://twitter.com/yoshi20190125/status/1596339507284283393

**Comment by myangelkamikaze** [comment]
Hmm, actually, one more random attempt at fixing it, could you try running 七四式 as admin?
![image](https://user-images.githubusercontent.com/40002167/204092718-e6d4f362-de7f-4be7-bb7d-551d39f50aa6.png)

**Comment by detsuku** [comment]
I tried it on another computer and it worked.
I changed the account and it didn't work.
It did not work even with administrator privileges.

Repeatedly uninstalling webviewr2 sometimes worked.
However, when I closed  7四式, it did not work the next time I started it.
Now it is working.
I hope this will work stably.
However, the difference between when it works and when it doesn't is not clear, so there is still an element of uncertainty.

**Comment by myangelkamikaze** [comment]
We added the option to use CefSharp instead of WebView2 in the latest preview, this might fix the problem you're having.
https://github.com/ElectronicObserverEN/ElectronicObserver/releases/tag/5.2.0-preview3

**Comment by detsuku** [comment]
The problem has not been reproduced since then.
I will continue to keep an eye on this.

I tried the 5.2.0-preview3 version and got the attached.
The cursor is not aligned with the screen and the email address field is not in the input state.
I pressed the Tab key several times and it became input state.
<img width="1280" alt="2022-12-12 (2)" src="https://user-images.githubusercontent.com/77409739/206950227-14ca2426-ae03-4cd7-973f-44d18a419df5.png">

**Comment by myangelkamikaze** [comment]
Can you rephrase, please? I don't understand what you mean.

**Comment by detsuku** [comment]
Only the upper left corner of the browser screen is displayed instead of the full screen.
At this time, the position of the mouse cursor does not match the position of the displayed item.
For example, clicking on the e-mail address field does not bring it into the input state.

**Comment by myangelkamikaze** [comment]
What's your display scaling? I think that might explain the cursor position being off (though it shouldn't happen either way).

**Comment by myangelkamikaze** [comment]
Just made a new preview, which might resolve the cursor problem.

**Comment by detsuku** [comment]
The screen setup is as attached.

I tried with the 5.2.0-preview4 version.
When I click on the email address field, it goes into the input state, but when I press the
key, but I could not enter it.
<img width="369" alt="スクリーンショット 2022-12-14 214936ディスプレイ環境" src="https://user-images.githubusercontent.com/77409739/207618605-039f78a4-0d00-4810-bd49-3e2b524facc0.png">

**Comment by myangelkamikaze** [comment]
Ah, so the mouse is clicking where it's supposed to, correct? Keyboard not working properly was reported and fixed already (fix will be available in the next preview).

**Comment by myangelkamikaze** [comment]
This is probably the same issue as #396.
